models/comments.go

```diff
@@ -18,11 +19,19 @@ func init() {
 	orm.RegisterModel(new(Comment))
 }
 
-func FormatTime(t time.Time) string {
-    // Format the time according to the desired format
-    formattedTime := t.Format("Jan. _2, 2006 03:04PM")
+func normalizeTime(t time.Time) time.Time {
+
```

Comment:
Additional space.

Reply:
> Additional space.

Can you elaborate lol

Reply:
i think he means the extra line of empty space lollll

Reply:
`go fmt` should resolve these issues.